manifests/install/windows.pp

```diff
@@ -43,12 +43,37 @@
   notice ("Puppet upgrade log file at ${_logfile}")
   debug ("Installing puppet from ${_msi_location}")
 
+  $_helpers = windows_native_path("${::env_temp_variable}/helpers.ps1")
+  file { $_helpers:
+    ensure  => file,
+    content => file('puppet_agent/helpers.ps1')
+  }
+
   $_installps1 = windows_native_path("${::env_temp_variable}/install_puppet.ps1")
   puppet_agent_upgrade_error { 'puppet_agent_upgrade_failure.log': }
   file { $_installps1:
     ensure  => file,
     content => file('puppet_agent/install_puppet.ps1')
   }
+
+  $_prerequisites_check = windows_native_path("${::env_temp_variable}/prerequisites_check.ps1")
+  file { $_prerequisites_check:
+    ensure  => file,
+    content => file('puppet_agent/prerequisites_check.ps1')
+  }
+
+  exec { 'prerequisites_check.ps1':
+    command => "${::system32}\\WindowsPowerShell\\v1.0\\powershell.exe \
+                  -NoProfile \
+                  -NoLogo \
+                  -NonInteractive \
+                  ${_prerequisites_check} ${::puppet_agent::_expected_package_version} ${_msi_location} ${_logfile}",
+    returns => 0,
```

Comment:
Is this needed? Per https://puppet.com/docs/puppet/5.5/types/exec.html#exec-attribute-returns, this defaults to 0.